src/viewer.ts

```diff
@@ -60,6 +58,8 @@ class Viewer {
         this.app = app;
 
         app.graphicsDevice.maxPixelRatio = window.devicePixelRatio;
+        // @ts-ignore
+        pc.Texture.defaultAnisotropy = app.graphicsDevice.maxAnisotropy;
```

Comment:
Maybe remove this line as we haven't decided to merge the engine side of this yet?

Reply:
Oh yes thanks, wasn't meant to be there.